test/runtests.jl

```diff
@@ -1,10 +1,81 @@
 using LinearAlgebra
+using SparseArrays
 using ProximalOperators
 using ShiftedProximalOperators
 using Test
 
 include("test_psvd.jl")
 
+for (op,composite_op,shifted_op) ∈ zip((:NormL1,), (:CompositeNormL1,), (:ShiftedCompositeNormL1,))
+  @testset "$shifted_op" begin
+    ShiftedOp = eval(shifted_op)
+    CompositeOp = eval(composite_op)
+
+    function c!(z,x)
+      z[1] = 2*x[1] - x[4]
+      z[2] = x[2] + x[3] 
+    end
+    function J!(z,x)
+      z .= sparse(Float64[2 0 0 -1;0 1 1 0])
+    end
+    λ = 3.62
+    Op = eval(op)
+    h = Op(λ)
+
+    b = zeros(Float64,2)
+    A = sparse(Matrix{Float64}(undef,2,4))
+
+    ψ = CompositeOp(λ,c!,J!,A,b)
+
+    # test non shifted operator
+    @test ψ(ones(Float64,4)) == h([1,2])
```

Comment:
Why does $\psi$ take an input of length 4 but $h$ takes an input of length 2 here?

Reply:
In our implementation, $\psi$ represents $\psi : \mathbb{R}^n \xrightarrow[]{} \mathbb{R}^m : s \xrightarrow[]{} ||c(x)+J(x)s||_1$. In this test, $n=4$ and $m = 2$.